omod/src/main/webapp/resources/scripts/controllers/requestAppointmentController.js

```diff
@@ -101,6 +101,7 @@ angular.module('appointmentscheduling.requestAppointment')
 
             AppointmentService.saveAppointmentRequest(appointmentRequestToSave).then(function() {
                 // success callback
+            	emr.successAlert("appointmentschedulingui.requestAppointment.added");
```

Comment:
Do you have this appointmentschedulingui.requestAppointment.added exisisting in your messages.properties file?

Reply:
Go on and assign the ticket to your self!

Reply:
@HerbertYiga thanks for the review I stand to be corrected but I thought this ticket needed to change the flow because to get the message to appear, you’ll need to set a property in Java. The property is the same properties set [in this controller](https://github.com/openmrs/openmrs-module-appointmentschedulingui/blob/ef5fb117f13806c21262e289fa86c3e9e772c11e/omod/src/main/java/org/openmrs/module/appointmentschedulingui/page/controller/SessionInfoPageController.java) also from the appointment stuff). Its  the controller for when a new appointment has been created and  not when a request has been submitted that is why I chose to redirect to a hard-coded location..

Reply:
@HerbertYiga am thinking of changing the emr.successAlert("appointmentschedulingui.requestAppointment.added");  to emr.successAlert("appointmentschedulingui.requestAppointment.added"); What is your thought.